spec/support/ansible_shared/automation_manager/configuration_script.rb

```diff
@@ -1,14 +1,27 @@
 require 'ansible_tower_client'
 require 'faraday'
 
-shared_examples_for "ansible configuration_script" do
+shared_examples_for "ansible configuration_script" do |embedded|
   let(:api)          { double(:api, :job_templates => double(:job_templates)) }
   let(:connection)   { double(:connection, :api => api) }
   let(:job)          { AnsibleTowerClient::Job.new(connection.api, "id" => 1) }
-  let(:job_template) { AnsibleTowerClient::JobTemplate.new(connection.api, "limit" => "", "id" => 1, "url" => "api/job_templates/1/", "name" => "template", "description" => "description", "extra_vars" => {:instance_ids => ['i-3434']}) }
-  let(:manager)      { FactoryGirl.create(:automation_manager_ansible_tower, :provider, :configuration_script) }
+  let(:job_template) do
+    AnsibleTowerClient::JobTemplate.new(connection.api,
+                                        "limit"       => "",
+                                        "id"          => 1,
+                                        "url"         => "api/job_templates/1/",
+                                        "name"        => "template",
+                                        "description" => "description",
+                                        "extra_vars"  => {:instance_ids => ['i-3434']})
+  end
+
+  let(:manager) do
+    FactoryGirl.create(embedded ? :embedded_automation_manager_ansible : :automation_manager_ansible_tower,
```

Comment:
yeah, that's why in the caller, I am using `it_behaves_like` as the doc says,
```
To prevent this kind of subtle error a warning is emitted if you declare multiple
methods with the same name in the same context. Should you get this warning
the simplest solution is to replace include_examples with it_behaves_like, in this
way method overriding is avoided because of the nested context created by it_behaves_like
```